src/components/project/NewProject/NewProject.tsx

```diff
@@ -170,13 +171,19 @@ const NewProject: FC<Props> = ({
       return;
     }
 
-    if (!importURL || !active) {
+    if (
+      (projects.length == 0 && !importURL && !active) ||
+      (projects.length !== 0 && (!importURL || !active))
```

Comment:
Please add a comment describing what these two possible cases stand for.

"If a new IDE without any previously opened projects is opened, and there is no URL to import, do not prepare New Project dialog for import"